tawazi/node/functions.py

```diff
@@ -0,0 +1,94 @@
+"""Helpers for node subpackage."""
+
+from typing import Any, Callable, Dict, Iterable, Iterator, List, Optional, Tuple, Union
+
+from . import node
+from .node import ReturnExecNode, UsageExecNode
+
+ReturnUXNsType = Union[
+    None, UsageExecNode, Tuple[UsageExecNode, ...], List[UsageExecNode], Dict[str, UsageExecNode]
+]
+
+
+def _wrap_in_iterator_helper(
```

Comment:
did you remove ruff on this file ? How can it pass without the docstrings ?

Reply:
variables / functions that have `_` at the start are private. No need to document them

Reply:
ah nice catch